### PR TITLE
WIP: mdr robocup tasks/find my mates

### PR DESCRIPTION
**Pull request by PatrickNa**
The aim is to define the "find my mates" scenario for the RoboCup@Home competition. This pull request addresses issue #105

### CODE REVIEW

File: mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,195 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mdr_find_my_mates)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```
Comment (minhnh) [inline mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt:4]: @argenos @alex-mitrevski what's the consensus for removing the auto-generated comments in  `CMakeLists.txt`? I personally prefer to remove them so the file is not too cluttered.

Comment (robertocaiwu) [reply mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt:4]: I'm also in favor of removing it.

Comment (alex-mitrevski) [reply mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt:4]: I always remove those.

Comment (PatrickNa) [reply mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt:4]: Will do at some point.

**Comment by PatrickNa** [comment]
The following figure visualizes the scenario up to this point:
find_my_mates

It can be noted that some of the actions are related with each other. For example the _request_actions can most probably originate from the same action with a different content.

The action _Localize Person_ includes the following sub-processes:
- Navigate around the area.
- Scan the surroundings looking for persons (#26 ).
- Save the location of the person.

The actions _Analyze Location_ and _Analyze Person_ refer to perception and the recognition of identifying objects.

**Pull request closed without merging**